### ISSUE TITLE
Copy-paste bug found by static code analysis tool

### ISSUE THREAD

**Issue by npapp** [webrtcHacks/adapter#851]
A static code analysis tool found a copy-paste bug that I want to share with you.

I don't really want to explain it, I made a quick fix instead, feel free to cherry-pick:
https://github.com/npapp/adapter/commit/dc9420708405723ba11712a5b3777f3d0439d345

**Comment by fippo** [comment]
can you make a PR please (which requires signing the CLA)? Ideally with just the change in src/js/safari/safari_shim.js

(I am also interested in how you found it... catching bugs is always good!)

**Comment by npapp** [comment]
Sorry, I was on holiday.

I was too lazy to read the CLA for this small change, that's why I wanted you to cherry-pick. :)

Oh the release folder is generated, should have noticed, my bad.

I ran a static code analyzer tool on a library which is dependant on webrtc-adapter npm package and browserified. It found this problem and I saw that it is not in the library's code but in the webrtc-adapter's so I decided to share it with you, that's how. :)
